runners/core-java/src/main/java/org/apache/beam/runners/core/SplittableParDoViaKeyedWorkItems.java

```diff
@@ -87,6 +89,12 @@
     public String getUrn() {
       return SplittableParDo.SPLITTABLE_GBKIKWI_URN;
     }
+
+    @Nullable
+    @Override
+    public RunnerApi.FunctionSpec getSpec() {
```

Comment:
<!--thread_id:cc_143604160_t; commit:a9a58870c39402cb77114fb054b85962a63a31a0; resolved:1-->
<!--section:context-quote-->
> **tgroh** wrote:
> Should this be NotSerializable?

<!--section:body-->
Done. UnsupportedOperationException here. Because this transform does not need a translator, it does not have one and uses the default RawPTransformTranslator to pull out the URN.